lib/matplotlib/artist.py

```diff
@@ -1296,34 +1274,25 @@ def get_setters(self):
         Get the attribute strings with setters for object.  e.g., for a line,
         return ``['markerfacecolor', 'linewidth', ....]``.
         """
-
         return [prop for prop, target in self._get_setters_and_targets()]
 
     def is_alias(self, o):
-        """
-        Return *True* if method object *o* is an alias for another
-        function.
-        """
+        """Return whether method object *o* is an alias for another method."""
         ds = o.__doc__
         if ds is None:
             return False
         return ds.startswith('Alias for ')
 
     def aliased_name(self, s):
         """
-        return 'PROPNAME or alias' if *s* has an alias, else return
-        PROPNAME.
+        return 'PROPNAME or alias' if *s* has an alias, else return PROPNAME.
 
         e.g., for the line markerfacecolor property, which has an
         alias, return 'markerfacecolor or mfc' and for the transform
         property, which does not, return 'transform'
         """
-
-        if s in self.aliasd:
-            return s + ''.join([' or %s' % x
-                                for x in sorted(self.aliasd[s])])
-        else:
-            return s
+        return s + ''.join(' or %s' % x
+                           for x in sorted(self.aliasd.get(s, [])))
```

Comment:
Just to keep this structurally similar to `alias_name_rest()`:
~~~
aliases = ''.join(' or %s' % x for x in sorted(self.aliasd.get(s, [])))
return s + aliases
~~~
Also slightly more readable.